pkg/api/entries.go

```diff
@@ -92,11 +93,32 @@ func logEntryFromLeaf(ctx context.Context, signer signature.Signer, tc TrillianC
 		return nil, fmt.Errorf("signing entry error: %w", err)
 	}
 
+	// sign a checkpoint as a commitment to the current root hash
+	sth, err := util.CreateSignedCheckpoint(util.Checkpoint{
```

Comment:
overall is there any way to abstract this to a func? i think it's called about 4 separate times in this code, all the same

Reply:
Done, moved to a new function (with tests!)